### PR TITLE
Add new SSD type support

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Description

Add new SSD vendor "Swissbit" support

#### Motivation and Context

Add new SSD vendor "Swissbit" support

#### How Has This Been Tested?

Unit test
Manual test

### CODE REVIEW

**Review by keboliu — DISMISSED** [review]

File: sonic_platform_base/sonic_ssd/ssd_generic.py [hunk 1]
```diff
@@ -154,6 +159,19 @@ def parse_virtium_info(self):
...
                     pass
 
+    def parse_swissbit_info(self):
+        if self.ssd_info:
+            health_raw = self.parse_id_number(SWISSBIT_HEALTH_ID)
+            if health_raw == NOT_AVAILABLE:
+                self.health = NOT_AVAILABLE
+            else:
+                self.health = health_raw.split()[-1]
+            temp_raw = self.parse_id_number(INNODISK_TEMPERATURE_ID)
```
Comment (prgeor) [inline sonic_platform_base/sonic_ssd/ssd_generic.py:169]: @Junchao-Mellanox this parsing seems to be for swissbit, why using innodisk id?

File: sonic_platform_base/sonic_ssd/ssd_generic.py [hunk 2]
```diff
@@ -23,6 +23,8 @@
 # Set Vendor Specific IDs
 INNODISK_HEALTH_ID = 169
 INNODISK_TEMPERATURE_ID = 194
+SWISSBIT_HEALTH_ID = 248
+SWISSBIT_TEMPERATURE_ID = 194
```
Comment (prgeor) [inline sonic_platform_base/sonic_ssd/ssd_generic.py:27]: @Junchao-Mellanox where is temp id used?

File: sonic_platform_base/sonic_ssd/ssd_generic.py (hunk 1 above)
Comment (Junchao-Mellanox) [reply sonic_platform_base/sonic_ssd/ssd_generic.py]: Fixed

File: sonic_platform_base/sonic_ssd/ssd_generic.py (hunk 2 above)
Comment (Junchao-Mellanox) [reply sonic_platform_base/sonic_ssd/ssd_generic.py:27]: Fixed

**Review by prgeor — APPROVED** [review]

**Review by keboliu — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Hi @StormLiangMS , could you please help cherry-pick to 202305?

**Comment by StormLiangMS** [comment]
@Junchao-Mellanox conflict with 202305, could you help to create separate PR?

**Comment by Junchao-Mellanox** [comment]
@StormLiangMS  PR for 202305 https://github.com/sonic-net/sonic-platform-common/pull/396

**Pull request merged**